core/plot.js

```diff
@@ -0,0 +1,141 @@
+            var currentValue = d.properties.count
+            return +currentValue
+        });
+    topo_cartogram.features(topo, topo.objects.tiles.geometries)
+    topo_cartogram.value(function (d) {
+        var currentValue = populationJson[d.properties.id][year]
+        return +currentValue
+    });
+
+    var topoFeatures = topo_cartogram(topo, topo.objects.tiles.geometries).features
+    export_format(topoFeatures);
+
+    let newHexbin = hexbin()
+        .radius(radius)
+        .x(function (d) { return d.x; })
+        .y(function (d) { return d.y; })
+
+    d3.select('#container').selectAll("*").remove()
+    const svg = d3.select('#container')
+        .append('svg')
+        .attr('width', width + margin.left + margin.top)
+        .attr('height', height + margin.top + margin.bottom)
+        .append('g')
+        .attr('transform', `translate(${margin.left} ${margin.top})`);
+
+    svg.append('g').attr('id', 'hexes')
+        .selectAll('.hex')
+        .data(getGridData(cellShape, newHexbin, pointGrid))
+        .enter().append('path')
+        .attr('class', 'hex')
+        .attr('transform', getTransformation(cellShape))
+        .attr('d', getPath(cellShape, newHexbin, shapeDistance))
+        .style('fill', '#fff')
+        .style('stroke', '#e0e0e0')
+        .style('stroke-width', strokeWidth)
+        .on("click", mclickBase);
+
+    let features = flatten_features(topoFeatures);
+    for (let i = 0; i < features.length; i++) {
+        for (let j = 0; j < features[i].coordinates.length; j++) {
+            var polygonPoints = features[i].coordinates[j];
+
+            let usPoints = pointGrid.reduce(function (arr, el) {
+                if (d3.polygonContains(polygonPoints, [el.x, el.y])) arr.push(el);
+                return arr;
+            }, [])
+
+            svg.append('g')
+                .attr('id', 'hexes')
+                .selectAll('.hex')
+                .data(getGridData(cellShape, newHexbin, usPoints))
+                .enter().append('path')
+                .attr('class', 'hex' + features[i].properties.id)
+                .attr('transform', getTransformation(cellShape))
+                .attr("x", function (d) { return d.x; })
+                .attr("y", function (d) { return d.y; })
+                .attr('d', getPath(cellShape, newHexbin, shapeDistance))
+                .style('fill', colors[i % 19])
+                .style('stroke', '#000')
+                .style('stroke-width', strokeWidth)
+                .on("click", mclick)
+                .on("mouseover", mover)
+                .on("mouseout", mout)
+                .call(d3.drag()
+                    .on("start", dragstarted)
+                    .on("drag", dragged)
+                    .on("end", dragended));
+        }
+    }
+}
+
+function getData(data) {
+    var obj = {}
+    for (var x in data) {
+        obj[data[x].code] = data[x]
+    }
+    return obj
+}
+
+function flatten_features(topoFeatures) {
```

Comment:
There are quite a few sneaky snake cases throughout the code. Might want to pick a style to stick with 😁

Reply:
Agreed, there were two other, following camel case everywhere from now on.